src/shipit_code_coverage_crawler/requirements.txt

```diff
@@ -1 +1,4 @@
 -e ./../../lib/cli_common[log] #egg=mozilla-cli-common
+
+https://github.com/marco-c/firefox-code-coverage/archive/e9073166968dd4ac33510a61116f8b8855b933af.tar.gz#egg=firefox-code-coverage
+https://github.com/mozilla/coverage-crawler/archive/b3884feffec33ca626836e7c410f6f9bcd701a2d.zip#egg=coverage-crawler
```

Comment:
you can point to master branch instead of revision since pypi2nix will (at least it should) pin to exact revision at the time you run pypi2nix command. That would be maybe easier to follow development of firefox-code-coverage and coverage-crawler.

Reply:
I prefer to be extra explicit here.